extra-utils/udiskie/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=udiskie
+PKGSEC=utils
+PKGDES="a udisks-2 GUI frontend"
+PKGDEP="python-3 docopt libappindicator pyyaml pygobject-3 libnotify udisks-2 polkit systemd python-keyutils"
+BUILDDEP="asciidoc"
+
+ABTYPE=python
+NOPYTHON2=1
```

Comment:
Does it actually not support Python 2?

Reply:
According to [udiskie's README](https://github.com/coldfix/udiskie/blob/master/README.rst), it has dropped Python2 supporting after 1.7.x

Reply:
Got it, thanks.